dashboard/app/views/pd/application/teacher_application_mailer/accepted_registration.html.haml

```diff
@@ -9,12 +9,11 @@
   = @application.effective_regional_partner_name + "'s"
   = @application.year
   Professional Learning Program for
-  = @application.course_name + ","
   - if @application.regional_partner
-    %strong at no cost to you or your school,
+    = @application.course_name
```

Comment:
Just checking: this looks right, right? The space comes from the line break?

ie this looks like "CSD in partnership with code.org" and not "CSDin partnership with code.org"

Reply:
Yes the space comes from the line break!